src/views/virtualmachines/actions/components/CloneVMModal/utils/helpers.tsx

```diff
@@ -0,0 +1,193 @@
+import * as React from 'react';
+import produce from 'immer';
+import { WritableDraft } from 'immer/dist/internal';
+
+import { ensurePath } from '@catalog/utils/WizardVMContext';
+import { V1beta1DataVolume } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { IoK8sApiCoreV1PersistentVolumeClaim } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import { V1DataVolumeTemplateSpec, V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import {
+  getDataVolumeTemplates,
+  getDisks,
+  getInterfaces,
+  getVolumes,
+} from '@kubevirt-utils/resources/vm';
+import { formatBytes } from '@kubevirt-utils/resources/vm/utils/disk/size';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+export const getName = (obj: K8sResourceCommon) => obj?.metadata?.name;
```

Comment:
no need for these helpers, either we already have them or you can just use `obj?.metadata?.name` directly

Reply:
+1 removed